app/models/competition.rb

```diff
@@ -189,6 +190,9 @@ class Competition < ApplicationRecord
   MAX_MARKDOWN_LENGTH = 255
   MAX_COMPETITOR_LIMIT = 5000
   MAX_GUEST_LIMIT = 100
+  NEWCOMER_MONTH_ENABLED = true
+  MAX_NEWCOMER_SPOTS_RESERVED_FRACTION = 0.5
```

Comment:
Basically, we should also change this to `MAX_NEWCOMER_MONTH_SPOTS_RESERVED_FRACTION`, but that becomes unfortunately long. Any ideas for a shorter name?

Reply:
Changed to `NEWCOMER_MONTH_RESERVATIONS_FRACTION` which isn't _much_ shorter but perhaps acceptably not-massive. My next suggestions would be just `NEWCOMER_MONTH_RESERVATIONS` (ie, drop the "fraction")